### ISSUE TITLE
Stress test excessive number of random delegations on stressnet

### ISSUE THREAD

**Issue by rlan35** [harmony-one/harmony#2502]
In the stress testnet,

1. create 100 validators
2. prepare 100k different addresses each funded with 100 tokens
3. for each of the 100k addresses, randomly delegate a certain number of tokens (1-10 randomly) to 1 of the 100 validators.

Do the profiling of CPU and memory on shard 0

**Comment by Daniel-VDM** [comment]
For 3-14-20, we launched STN on commit: `a6cf930db67f8cd7ae3eb2aa891c386129cfa9cd`.

2 shards, 10 nodes, 3 keys per node. 

The current expectation is to refresh STN soon, so maybe @SebastianJ can try this test at a smaller scale like maybe one 10th of the scale.

**Comment by SebastianJ** [comment]
~105k delegations test performed - see separate stress test spreadsheet.

TLDR; shard 0 lost consensus, seems likely that Ethereum's RLP encoder uses a lot of memory and CPU when encoding large arrays/slices (which slows down block latency up to 45-50s, eventually resulting in consensus being lost).
